exporter/influxdbexporter/writer.go

```diff
@@ -49,7 +49,7 @@ func newInfluxHTTPWriter(logger common.Logger, config *Config, telemetrySettings
 		encoderPool: sync.Pool{
 			New: func() interface{} {
 				e := new(lineprotocol.Encoder)
-				e.SetLax(false)
+				e.SetLax(true)
```

Comment:
Out of ignorance: What's the level of concern here for unexpected consequences of allowing unvalidated data to be sent?

Reply:
Minimal. If the receiver (likely InfluxDB or Telegraf) would reject with 4XX.

It's possible to write invalid data in strict mode as well, of course. Schema conflicts, for example.

Reply:
Good to know, thank you!